lib/src/chart/radar_chart/radar_chart_data.dart

```diff
@@ -160,12 +162,19 @@ class RadarChartData extends BaseChartData with EquatableMixin {
   /// If [isMinValueAtCenter] is true, the minimum value of the [RadarChart] will be at the center of the chart.
   final bool isMinValueAtCenter;
 
+  /// [max] and [min] are used to define the maximum and minimum values of the [RadarChart]
+  final RadarEntry? max;
```

Comment:
Doesn't it make sense to just have a double value for the `min` and `max` properties?
(Just like what we have for LineChart and BarChart)